src/styles/components/_product-media.scss

```
@@ -0,0 +1,30 @@
+.product-media {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: rem(16px);
+  box-sizing: border-box;
+
+  &__header {
+    margin-bottom: rem(24px);
+  }
+
+  &__title {
+    margin: 0;
+  }
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: rem(200px);
+    width: rem(200px);
```

If we want to completely follow this (ref)[https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/], please put `width` before `height`
```suggestion
    width: rem(200px);
    height: rem(200px);
```